src/pages/markets/_markets_accordion.js

```diff
@@ -58,13 +58,16 @@ const MarketsAccordion = ({ renderTitle, renderDetails }) => {
                 </Button>
             </Title>
             {is_show_detail && (
-                <Details background="var(--color-grey-23)">{renderDetails()}</Details>
+                <Details background="var(--color-grey-23)">
+                    {args ? renderDetails(args) : renderDetails()}
+                </Details>
             )}
         </Wrapper>
     )
 }
 
 MarketsAccordion.propTypes = {
+    args: PropTypes.any,
```

Comment:
what's args here? 🤔

Reply:
`args` is used to custom display `details content` where one of the details is a subset of another.
Instead of creating different Details component for each one we can use `args` to display the subset needed. 
I guess it's better to rename it to be more descriptive